examples/wrapper-types/uniffi.toml

```diff
@@ -9,3 +9,30 @@ cdylib_name = "wrapper_types"
 
 [bindings.kotlin]
 cdylib_name = "wrapper_types"
+
+# Python wrapper implementation for JsonObject
+[wrappers.JsonObject.python]
+
+code = """
+class JsonWrapper:
+    fn from_wrapped(string):
+        json.loads(string)
+
+    fn into_wrapped(obj):
+        json.dumps(obj)
+}
+"""
+imports = ["json"]
+wrapper_class = "JsonWrapper()"
```

Comment:
In the last proposal you specified the wrapping/unwrapping functions.  I like specifying an object to handle this for a few reasons:
  - It's how the scaffolding side works.
  - It keeps the concept of an "extension type" alive.  We could share a `jsonwrapper.py` file between projects to avoid rewriting the same code.
  - It would work better if we ever needed to add extra methods or arguments to these methods.

Reply:
My problem with your proposal above is that it's a class that exists only as a holder for the 2 methods (which should be marked as `@static`? A downside is that it gives the impression that this `JsonWrapper` is actually going to be used by the consumer - ie, that a casual reader would assume that anything declared in the UDL would take/get `JsonObject`, but that's not true. So `JsonObject` should probably be called something like `JsonConverter`?

But it still seems overkill - `json.loads({})` and `json.dumps(})` is significantly less boilerplate and has identical semantics. So I'd be inclined to simplify this out to:

```
setup = "import json"
from_primitive = "json.loads({})
to_primitive = "json.dumps({})"
```

but if someone really did think the wrapper added value, they could still write:
```
setup = """
import json # or keep 'imports', but still not sold on that ;)
class JsonWrapper:
    @staticmethod
    fn from_prim(string):
        json.loads(string)
    @staticmethod
    fn into_prim(obj):
        json.dumps(obj)
}
from_primitive = "JsonWrapper.from_prim({})"
to_primitive = "JsonWrapper.to_prim({})"
```

which IIUC is identical to your example stylistically, and identical to both the above semantically.

(Not too bothered by this though)